sale_attached_product/README.rst

```diff
@@ -0,0 +1,114 @@
+==========================
+Attached products in sales
+==========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/sale-workflow/tree/13.0/sale_attached_product
+    :alt: OCA/sale-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-workflow-13-0/sale-workflow-13-0-sale_attached_product
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/167/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to define a set of products which will be added atomatically to the
+sales order whenever that product is present on it.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure attached products:
+
+#. Go to *Sales > Catalog > Products* and choose on you want to attach products to.
```

Comment:
I cannot see the menu Catalog? [sale_attached_product_instance url](http://oca-sale-workflow-15-0-pr2393-941fc466917e.runboat.odoo-community.org/web?debug=1#cids=1&menu_id=331&action=505&model=sale.order&view_type=list)

![image](https://user-images.githubusercontent.com/19970138/222689938-19fb32b8-3603-4cd2-84fe-33b9e1682355.png)

Do I do something wrong?

Reply:
I update the CONFIGURE.rst file